metadata.yaml

```diff
@@ -8,4 +8,10 @@ tags:
 requires:
   reverseproxy:
     interface: http
-series: ['xenial']
\ No newline at end of file
+resources:
+    dhparam:
+        type: file
+        filename: dhparam.pem
```

Comment:
Hm, can the easy-rsa charm generate dh params for you? I'm generating dh params in the openvpn charms without any issue, so let's sit together about this if easy-rsa can't fix it.